framework/di/InvalidConfigException.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+
+namespace yii\di;
+
+use \yii\base\InvalidConfigException as Base;
```

Comment:
using Base as alias is not preferable. Using the full class name is better here

Reply:
Why not implement `ContainerExceptionInterface` at `yii\base\InvalidConfigException` after all, the `init()` method on most classes throws that exception already.

Reply:
`yii\base\InvalidConfigException` has a wider meaning - it refers to **any** mis-configuration error, which is not mandatory triggered by DI.

Reply:
So? The PSR never says that only DI can throw said exceptions.

Besides as you say any configuration error will throw `yii\base\InvalidConfigException` which by the PSR.

> A call to the get method with a non-existing id MUST throw a Psr\Container\NotFoundExceptionInterface.

So `$di->get('db', [/* invalid params or config */)` must throw `Psr\Container\NotFoundExceptionInterface`